src/vmm/src/devices/virtio/vsock/mod.rs

```diff
@@ -18,6 +18,7 @@ mod packet;
 pub mod persist;
 pub mod test_utils;
 mod unix;
+pub mod vsock_metrics;
```

Comment:
typo in the `refactor(metrics): move vsock metrics out of logger` commit message: `vosck`

Reply:
done. Thanks!